src/use_case/create_announcement/CreateAnnouncementDataAccessInterface.java

```diff
@@ -1,10 +1,9 @@
 package use_case.create_announcement;
 
 
-import entity.Announcement;
+import entity.CommonAnnouncement;
 
 public interface CreateAnnouncementDataAccessInterface {
-    boolean existsByName(String announcementTitle);
 
-    void save(Announcement announcement);
```

Comment:
Design pattern question:
since we have different types of entities to "save", do you think it's better to just create a facade save?